.linkml-lint-config.yml

```diff
@@ -0,0 +1,4 @@
+extends: recommended
+rules:
+  standard_naming:
+    level: disabled
```

Comment:
Why are you disabling the `standard_naming` rule recommended by the linkml linter? we are following that naming pattern in MIxS.

Reply:
Not for enums! I added this at the suggestion from @turbomam as he said it's likely that 'that ship has sailed' (direct quote) ;)

See the linting results below.

[lint-results.zip](https://github.com/user-attachments/files/18598969/lint-results.zip)

Reply:
Ok, I'll remove those parts. The linting results are from the latest comment on of the gsc mix schema mind.

But we can see when the action is working.